scripts/cloneAndPatch.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+basedir=$(dirname "$(realpath "$0")")
+graal_version=$1
+major_minor_graal_version=$(cut -d '.' -f 1,2 <<< $graal_version)
+graal_repo="https://github.com/oracle/graal.git"
+graal_branch="release/graal-vm/$major_minor_graal_version"
```

Comment:
We should be checking out the tag, not the release branch.

Reply:
Oh I see you switch to the tag later on. Is it not possible to checkout the tag directly?

Reply:
How do you do that?

Reply:
> 

No. It is not possible to checkout a tag directly. I have tried to do that but we cannot reference a tag. Only a branch. Even though, people confuse tag with branches when they try to explain the checkout process.

The best way is to download the commits and then switch to the tag that we want. That is why the option "--filter" is there. To avoid downloading everything related to git.